### DISCUSSION TITLE
Stay Profile Buttons

### DISCUSSION THREAD

**Discussion by toadee** [hawky358/hass_ids_hyyp#29] [Q&A] [answered]
Hi,

I'm trying to add a custom alarm panel card and I see in all examples there is a button for stay profiles however in my entities list, I do not have any stay profile buttons, not even for the OOB profile which is currently active on my HYYP app.

Are these buttons supposed to pull through after adding the integration or is there additional settings I need to do in HA?

Thanks

**Comment by hawky358** [comment]
Hello,

No additional configuration is required.

If you have a stay profile configured on your panel, then you should have a button entity called `button.[site_name]_[partition_name]_[stay_profile_name]`. 
It will also appear in the "Controls" section of the IDS "Device".
You can find the detail in the [Readme ](https://github.com/hawky358/hass_ids_hyyp?tab=readme-ov-file#features)under "Multiple Stay Profiles"

Nothing seems to have changed on the IDS server side, so it should all still be the same.

I don't know what OOB means, but I assume it's a personal acronym of some kind for your stay profile.

**Reply by toadee** [reply]
OOB - Out of Box. Essentially the Generic stay profile when the system is installed, but even that is not pulling through.

Have reached out to the security installers and they recommend I reach out to IDS support as it may be an issue related to a firmware update requirement. Will do that first before returning back here

**Reply by hawky358** [reply]
It's unlikely that it's a firmware issue.  If it appears on the app then it's being sent to IDS servers, which means it should be visible to the integration. It gets the exact information that the app gets.

There is no "generic stay profile" as such which comes with the panel. It needs to be configured.  
Did you assign zones to the stay profile?
On which partition is the profile?

**Reply by toadee** [reply, marked as answer]
I did assign zones to each profile on partition 1.

IDS walked me through manual syncing through the keypad (since only 1 of the 3 stay profiles were showing). 

Last ditch effort was to disconnect battery, turn power off and turn back on and that has somehow fixed everything. Stay profiles now showing on app and on integration.

**Reply by hawky358** [reply]
Good to hear you got it sorted.😊

Major changes on the panel does often take some time to update to ids servers since the panel doesn't continuously communicate with ids.

This includes zone and partition names for example.
I guess it does a full upload to ids at boot👍🏻
